backend/tests/v5/unit/models/revisions.test.js

```diff
@@ -53,7 +53,54 @@ const testGetLatestRevision = () => {
 	});
 };
 
+const testGetRevisions = () => {
+	describe('GetRevisions', () => {
+		const expectedData = [
+			{ _id: 1, author: 'someUser', timestamp: new Date() },
+			{ _id: 2, author: 'someUser', timestamp: new Date() },
+			{ _id: 3, author: 'someUser', timestamp: new Date(), void: true },
+		];
+
+		test('Should return all container revisions', async () => {
+			jest.spyOn(db, 'find').mockResolvedValue(expectedData);
+			const res = await Revisions.getRevisions('someTS', 'someModel', true);
```

Comment:
check the querys we're passing through.

just like the ones I did in favourites: https://github.com/3drepo/3drepo.io/blob/ISSUE_2711/backend/tests/v5/unit/models/users.test.js#L126